flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdHandlerTestBase.scala

```diff
@@ -749,7 +749,7 @@ class FlinkRelMdHandlerTestBase {
       false,
       false,
       false,
-      Seq(Integer.valueOf(0)).toList,
+      Seq(Integer.valueOf(3)).toList,
```

Comment:
What's going on here?

Reply:
I think this test was not implemented correctly. I got a type mismatch here. The purpose of this test is about propagating metadata, so the change should not affect the test negatively.